demo/index.module.scss

```diff
@@ -0,0 +1,142 @@
+body {
+  margin: 0;
+  padding: 0.5em;
+}
+
+.container {
+  height: 100vh;
+  font-family: ReithSerif, Fallback, sans-serif;
```

Comment:
I don't think `Fallback` will resolve to anything - is this meant to be a reference to a mixin?

Reply:
Good catch - I followed this from somewhere (I'm sure int-gel-matter or a component library) but I can't find it anymore - it's only us doing it!

So yeah - I think this can be removed 😅